docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,544 @@
+
+The diagram shows the new ordering node that has been added as a follower:
+
+![create-channel.osnadmin4](./osnadmin4.png)
+_OSN4 is added to the channel as a follower. Its status is "onboarding" while its ledger catches up to the config block provided on the osnadmin channel join command._
+
+
+Although the orderer status transitions from **onboarding** to **active**, it is still unable to participate in the ordering service because its `consensusRelation` status is **follower**, but it will continue to pull blocks and stay current with the ledger. After the status becomes **active** and you are ready for the orderer to transition from a follower to a consenter, any orderer organization admin can add the orderer to the consenter set on the channel by submitting a [channel configuration update transaction](../config_update.html), or by using the [fabric-config](https://github.com/hyperledger/fabric-config) library.
+
+**Note:** you should never attempt any configuration changes to the Raft consenters, such as adding a consenter, unless all consenters are online and healthy, as you risk losing quorum and therefore the ability to process transactions.
+
+Ensure that the orderer organization has write privileges in the channel configuration so the orderer can become a consenter. When the channel update transaction is complete, you can use the `osnadmin channel list` command again to confirm that the `consensusRelation` status for the orderer automatically changes to **consenter**.
+
+```
+{
+  "name": "channel1",
+  "url": "participation/v1/channels/channel1",
+  "status": "active",
+  "consensusRelation": "consenter",
+  "height": 1152
+}
+```
+
+## Next steps
+
+### Join peers to the channel
+
+After the channel has been created, you can follow the normal process to join peers to the channel and configure anchor peers.  If the peer organization was not originally included in the channel configuration, you need to submit a [channel configuration update transaction](../config_update.html) to add the organization. If it did include the peer organization, then all peer organizations that are members of the channel can fetch the channel genesis block from the ordering service using the [peer channel fetch](../commands/peerchannel.html#peer-channel-fetch) command. The organization can then use the genesis block to join the peer to the channel using the [peer channel join](../commands/peerchannel.html#peer-channel-join) command. Once the peer is joined to the channel, the peer will build the blockchain ledger by retrieving the other blocks on the channel from the ordering service.
+
+### Add or remove orderers from existing channels
+
+You can continue to use the `osnadmin channel join` and `osnadmin channel remove` commands to add and remove orderers on each channel according to your business needs.
```

Comment:
I think it is important to mention that it is recommended to configure channel `A` and remove `OSNx` from the consenters set, prior to deleting channel `A` from orderer `OSNx`.

Reply:
You don't really need to do this, it is recommended. If the channel has fault tolerance, removing a channel from an orderer is equivalent to a single failure.